spec/features/questions/update_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+require 'pry'
+
+RSpec.describe "Update a question" do
+  context "Question exists" do
+    it "shows the question" do
+      question = RatingQuestion.create!(title: "hola")
+      visit "/rating_questions/#{question["id"]}/edit"
```

Comment:
The `question` object here is an instance of the `RatingQuestion` class -- a model -- and so you can get to its `id` attribute (or any attribute) by calling the method: `question.id`.